week1_gp6_zad2.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+using namespace std;
+unsigned int SIZE=100;
+
+struct OrderedPair{
+    int first;
+    int second;
+};
+
+void PrintOrderedPair(const OrderedPair& PairToPrint){
```

Comment:
It'll be nice if you're more consistent when naming your variables - you better pick a single [naming convention](https://en.wikipedia.org/wiki/Naming_convention_(programming)) and stick to it :P